### PR TITLE
Update readme: tutorial article and contribute section

### PR DESCRIPTION
**Pull request by KrzysztofWrobel**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -18,8 +18,7 @@ make_router(
 TODO We would like to have a nice graphic explaning routing mechanism
 -->
 
-[Live demo can be found here.](http://demo.appsilondatascience.com/shinyrouter/)
-
+[Live demo can be found here](http://demo.appsilondatascience.com/shinyrouter/) and a simple tutorial article is available on [Appsilon Data Science blog](http://blog.appsilondatascience.com/rstats/2016/12/08/shinyrouter.html).
```
Comment (filipstachura) [inline README.Rmd]: basic

File: README.Rmd [hunk 2]
```diff
@@ -70,7 +69,15 @@ server <- shinyServer(function(input, output) {
...
 ```
 
-Troubleshooting
+How to contribute?
+-----------------
+If you want to contribute to this project please submit a regular PR, once you're done with new feature or bug fix.<br>
+
+**Changes in documentation**
+
+Both repository **README.md** file and an official project documentation page are generated with Rmarkdown, so if there is a need to update them, please modify accordingly a **README.Rmd** file  and run a **build_readme.R** script to compile it.
```
Comment (filipstachura) [inline README.Rmd]: remove 'project'

File: README.Rmd (hunk 1 above)
Comment (KrzysztofWrobel) [inline README.Rmd]: OK

File: README.Rmd (hunk 2 above)
Comment (KrzysztofWrobel) [inline README.Rmd]: ok

**Pull request merged**